CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -368,7 +368,7 @@ public Signing(string loopName)
 
             public byte[] GetPublicKey() => _rsaWriter.ExportCspBlob(false);
 
-            public byte[] GetSignature(byte[] data) => _rsaWriter.SignData(data, new SHA1CryptoServiceProvider());
+            public byte[] GetSignature(byte[] data) => _rsaWriter.SignData(data, SHA1.Create());

```

Comment:
Is this change simply related to the dotnet update? It looks like the to functions have similar functionality.

Reply:
yes, its part of the breaking changes. They have flagged that obsoleted and ask to use that base class's Create method. I don't know when they introduced that base class or if it was always there